frontend/src/views/auth/Signin.vue

```diff
@@ -190,17 +183,13 @@ export default {
         router.push({ name: "auth.signup", replace: true });
       }
 
+      window.addEventListener("bb.oauth.signin", eventListener, false);
```

Comment:
nit: keep the original order (and normally we would put addEventListener at last to make it consistent with other places).